.circleci/config.yml

```diff
@@ -640,10 +651,12 @@ jobs:
           title: "selected subset"
           test_targets: "
             other.test_deterministic
+            core1
```

Comment:
The idea with `test-node-compat` was that we do just a small selection of tests.. rather than the entire core test suite like this.